cmd/util/datadir/report.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019-2023 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package datadir
+
+// TODO: eliminate duplication (copied from cmd/algocfg/report.go)
```

Comment:
I personally prefer TODOs to be captured in issues and not inline, but I've seen other people's commits w/ them left in as well so I don't think there's a consensus on this.

Reply:
> I personally prefer TODOs to be captured in issues

Agree,. This one id mostly a marker for myself (immediate next step after merge).